x-pack/plugins/siem/public/store/timeline/epic.ts

```diff
@@ -286,6 +293,7 @@ const timelineInput: TimelineInput = {
   dateRange: null,
   savedQueryId: null,
   sort: null,
+  timelineType: null,
```

Comment:
👍 I found a bug in my `create template timeline` PR, it is after updating a `template` timeline, it becomes a  normal timeline as I didn't add 
```
templateTimelineId: null,
templateTimelineVersion: null,
timelineType: null,
```
also after I adding these, I still get a `templateTimelineVersion conflict`, therefore I'd like to update my api to allow sending the `same` templateTimelineVersion, and I'll bump the version from api.